### PR TITLE
feat: add token table loading state

### PR DESCRIPTION
**Pull request by kayleegeorge**
feat: add loading state for token table when data is still being fetched 

Screen Shot 2022-06-28 at 1 58 30 PM

### CODE REVIEW

File: src/components/Explore/TokenTable.tsx [hunk 1]
```diff
@@ -25,7 +25,17 @@ export default function TokenTable() {
   const { data, error, loading } = useTopTokens()
   const timePeriod = TimePeriod.day
   if (loading) {
-    return <GridContainer>Top Token Data Loading</GridContainer>
+    const loadingRows = Array(10)
```
Comment (vm) [inline src/components/Explore/TokenTable.tsx:28]: i think you can make this once as a constant, or memoize this?

Comment (vm) [reply src/components/Explore/TokenTable.tsx:28]: since otherwise you create this every render

File: src/components/Explore/TokenRow.tsx [hunk 2]
```diff
@@ -207,6 +236,39 @@ function HeaderCell({
   return <Trans>{getHeaderDisplay(category, timeframe)}</Trans>
 }
 
+export function LoadingRow({ key }: { key: string }) {
+  return (
```
Comment (vm) [inline src/components/Explore/TokenRow.tsx:240]: do you think we should do this inline or make a separate row? my opinion is do it inline in the main row

Comment (vm) [reply src/components/Explore/TokenRow.tsx:240]: another option would be to make a `Row` component and then have `LoadedRow` and `LoadingRow` that use it. then the logic isn't coupled but also is abstracted

File: src/components/Explore/TokenRow.tsx [hunk 3]
```diff
@@ -174,6 +174,35 @@ const VolumeCell = styled(Cell)`
...
+const IconLoadingBubble = styled(LoadingBubble)`
+  border-radius: 50%;
+  width: 24px;
+`
+
+const SparkLineLoadingBubble = styled(LongLoadingBubble)`
+  height: 4px;
+`
+/*
+function LoadingBubble({ width, height }: { width: number; height: number }) {
```
Comment (vm) [inline src/components/Explore/TokenRow.tsx:201]: rm

File: src/components/Explore/TokenRow.tsx (hunk 2 above)
Comment (kayleegeorge) [reply src/components/Explore/TokenRow.tsx:240]: tokenRow, LoadedRow, and LoadingRow

File: src/components/Explore/TokenRow.tsx [hunk 4]
```diff
@@ -207,61 +230,128 @@ function HeaderCell({
...
+  price: JSX.Element | string
+  percentChange: JSX.Element
+  marketCap: JSX.Element | string
+  volume: JSX.Element | string
+  sparkLine: JSX.Element | null
+  swap: JSX.Element | null
+}) {
+  const headerRow = header ? 'header' : undefined
+  return (
+    <TokenRowWrapper key={key} className={headerRow}>
```
Comment (vm) [inline src/components/Explore/TokenRow.tsx:261]: make header a prop

File: src/components/Explore/TokenRow.tsx [hunk 5]
```diff
@@ -207,61 +230,128 @@ function HeaderCell({
...
+  price,
+  percentChange,
+  marketCap,
+  volume,
+  sparkLine,
+  swap,
+}: {
+  header: boolean
+  key: string
+  favorited: JSX.Element | null
```
Comment (vm) [inline src/components/Explore/TokenRow.tsx:249]: ReactNode for all these

**Pull request merged**